<?xml version="1.0"?>

<launch>
	<arg name="robotName" value="bebop"/>
	<!-- SLOW -->
	<!-- arg name="kp" default="3.000"/>
	<arg name="ki" default="1.000"/>
	<arg name="kd" default="0.001"/ -->
	<!-- FAST -->
	<arg name="kp" default="4.300"/>
	<arg name="ki" default="0.001"/>
	<arg name="kd" default="2.400"/>
	<arg name="ko" default="1.000"/>

	<remap from="/uav/command_velocity" to="/$(arg robotName)/command_velocity"/>
	<remap from="/uav/command/motor_speed" to="/$(arg robotName)/command/motor_speed"/>
	<remap from="/uav/position" to="/$(arg robotName)/position"/>
	<remap from="/uav/velocity" to="/$(arg robotName)/linear_velocity"/>
	<remap from="/uav/noise/position" to="/$(arg robotName)/noise/position"/>
	<remap from="/uav/odometry" to="/$(arg robotName)/odometry"/>
	<remap from="/uav/true_odometry" to="/$(arg robotName)/ground_truth/odometry"/>
	<remap from="/uav/trajectory" to="/$(arg robotName)/trajectory"/>
	<remap from="/uav/trajectory_velocity" to="/$(arg robotName)/trajectory_velocity"/>

	<node name="pid" pkg="controllers" type="pid" output="screen" args="$(arg kp) $(arg ki) $(arg kd)"/>
	<node name="trajectory" pkg="controllers" type="trajectory" output="screen" args=""/>
	<!--node name="logger" pkg="controllers" type="logger" output="screen" args="1-fast-pid-$(arg kp)-$(arg ki)-$(arg kd)"/ -->
	<node name="logger" pkg="controllers" type="logger" output="screen" args="2016.09.29"/>

	<!-- node name="dynamic_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" args=""/ -->
</launch>
